railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -55,10 +55,9 @@ group :development do
   # Note: Interferes with etag cache testing. Can be configured to work on production: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/README.md
   # gem "rack-mini-profiler", "~> 2.0"
 <%- end -%>
-<% if spring_install? -%>
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem "spring"
-<% end -%>
+  # Speed up rails commands in dev on slow machines / big apps with Spring.
+  # Run bundle exec spring binstub --all after uncommenting. See: https://github.com/rails/spring
```

Comment:
"Run bundle exec spring binstub --all after uncommenting." is also mentioned at https://github.com/rails/spring#setup, is it worth duplicating here?